web/src/site-admin/SiteAdminSidebar.tsx

```diff
@@ -46,7 +46,7 @@ export const SiteAdminSidebar: React.FunctionComponent<SiteAdminSidebarProps> =
 
         <Link to="/api/console" className={SIDEBAR_BUTTON_CLASS}>
             <ConsoleIcon className="icon-inline" />
-            API console
+            <span className="api-console-text">API console</span>
```

Comment:
In the sidebar headers, we actually just use a space:

https://github.com/sourcegraph/sourcegraph/blob/8bf8dac82992b00642933d647c46e8828d973529/web/src/components/Sidebar.tsx#L33

I would try that method first (the space was missing here, as they get collapsed if there is a newline).
```suggestion
            {' '}API console
```
(this needs formatting with Prettier after applying)